django_app_server_db_server/deployment/roles/app_server/tasks/main.yml

```diff
@@ -49,6 +52,11 @@
 
 - name: Rub out old eclaim_revamp folder
   file: path=/opt/eclaim_revamp state=absent
+  when: app_version == "eclaim_revamp"
+
+- name: Rub out old eclaim folder
```

Comment:
You sure you want to rub it out ? You really don't have to here. I put the rub out because I just wanted the most updated code from the repo and there are many changes (compared to old eclaim).